_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -373,7 +352,7 @@ mfxStatus VideoDECODEH265::Init(mfxVideoParam *par)
     if (m_platform != m_core->GetPlatformType())
```

Comment:
we have ```MFX_CHECK_WITH_ASSERT``` you can replace this whole block with this macro

Reply:
MFX_CHECK_WITH_ASSERT has one condition for assert and check, but they are different here

Reply:
oh, I see